qemu/tests/sev_hotplug_mem.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from avocado import TestError
+from virttest import error_context
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class MemoryHotplugSimple(MemoryHotplugTest):
+
+    def get_mem_by_name(self, vm, name):
+        """
+        Return memory object and pc-dimm devices by given name
+        """
+        dev_ids = map(lambda x: "-".join([x, name]), ["mem", "dimm"])
+        devices = filter(None, map(vm.devices.get_by_qid, dev_ids))
+        return [_[0] for _ in devices]
+
+    def unplug_memory(self, vm, target_mem):
+        """Unplug the target memory, if the memory not exists,
+           hotplug it, then unplug it
+        """
+        devs = self.get_mem_by_name(vm, target_mem)
+        if not devs and self.params.get("strict_check") != "yes":
+            self.hotplug_memory(vm, target_mem)
+            self.check_memory(vm)
+        return super(MemoryHotplugSimple, self).unplug_memory(vm, target_mem)
+
+    def check_memory(self, vm):
```

Comment:
We might need to overwrite the class method get_guest_total_mem instead of this function, e.g. 

@classmethod
def get_guest_total_mem(cls, vm):
      return rom_size + super().get_guest_total_mem(vm)

The we follow the original check_memory's steps to do the check

Reply:
Thanks Zhenchao, I didn't find there is an available function in VT.

Reply:
Hi Zixi, not sure if I got your point here, if you mean get_guest_total_mem function,
it's defined in MemoryBaseTest class, which is the parent class of MemoryHotplugTest

Reply:
Update as we discussed, thanks for reviewing!